### DISCUSSION TITLE
Replace options with commands?

### DISCUSSION THREAD

**Discussion by Kinrany** [gitext-rs/git-stack#304] [Ideas]
I find it surprising that `--push`, `--pull`, `--rebase` are options, not commands. They are mutually exclusive, are they not? Commands are commonly used to express this, and this applies to git itself.

**Comment by epage** [comment]
They are not mutually exclusive but we have been re-working the UX into individual commands
- `git stack sync`
- `git stack reword`
- `git stack amend`
- ...

You can run `git stack alias --register` to modify your config so you can run these as `git sync`, `git reword`, etc.
